src/io.F

```diff
@@ -277,14 +277,11 @@ subroutine cem_out
            call vtk_swap_field( Ur, Ui, Ui,vtkout1)
            call vtk_swap_field( Qr(1,1),Qr(1,2),Qr(1,3),vtkout2)
            call vtk_swap_field( Qi(1,1),Qi(1,2),Qi(1,3),vtkout3)
-           call vtk_swap_field( sUr,sQr(1,1),sQi(1,1),vtkout4)
 #elif defined DRIFT
            call vtk_swap_field( cN, cP, potent,vtkout1)
 #elif defined MAXWELL
            call vtk_swap_field(EN(1,1),EN(1,2),EN(1,3),vtkout1)
            call vtk_swap_field(HN(1,1),HN(1,2),HN(1,3),vtkout2)
-           call vtk_swap_field(sEN(1,1),sEN(1,2),sEN(1,3),vtkout3)
```

Comment:
For restart, we need only numeric solution (this also reduces file sizes). Analytics are from usersol.